kubernetes/base/stateful-set.yaml

```diff
@@ -1,51 +1,37 @@
 apiVersion: apps/v1
-kind: Deployment
-
+kind: StatefulSet
 metadata:
   name: chat-service
   labels:
     app: chat-service
-
 spec:
-  replicas: 1
-
+  replicas: 3
+  serviceName: $(SVC_NAME_INTERNAL)
```

Comment:
Not sure it's really worth breaking out the variable spec of kustomize to avoid duplication in two places.

Reply:
Made the name of the service a variable in response to Meidad's feedback on https://skillz.slack.com/archives/G0HECCC0P/p1571242223029400?thread_ts=1571177962.029100&cid=G0HECCC0P ( did not like the hardcoded chat-service-internal in there ).